plugins/modules/aci_maintenance_policy.py

```diff
@@ -218,16 +277,33 @@
 from ansible.module_utils.basic import AnsibleModule
 
 
+MATCH_RUN_MODE_MAPPING = dict(pause_always_between_sets="pauseAlwaysBetweenSets", pause_only_on_failures="pauseOnlyOnFailures", pause_never="pauseNever")
+MATCH_NOTIFY_CONDITION_MAPPING = dict(
+    notify_always_between_sets="notifyAlwaysBetweenSets", notify_never="notifyNever", notify_only_on_failures="notifyOnlyOnFailures"
+)
+MATCH_SMU_OPERATION_MAPPING = dict(smu_install="smuInstall", smu_uninstall="smuUninstall")
+MATCH_SMU_OPERATION_FLAGS_MAPPING = dict(smu_reload_immediate="smuReloadImmediate", smu_reload_skip="smuReloadSkip")
+
```

Comment:
move to constants file, then when other modules would have the same list they can be reused